open_by_laws/urls.py

```diff
@@ -0,0 +1,23 @@
+from django.urls import include, path
+
+from open_by_laws.views import HomePageView, MunicipalByLawsView
+
+urlpatterns = [
+    path("", HomePageView.as_view(), name="home_page"),
```

Comment:
We'll need a URL to redirect from the old place codes, such as `za-xxx/yyy` to `bylaws/xxx`

Reply:
We'll also need to redirect the by-law detail page so that we don't lose old links:

eg:

```
/za-cpt/act/by-law/2016/air-quality-management/eng/
->
/akn/za-cpt/act/by-law/2016/air-quality-management/
```

I suggest: `/za-<str>/act/by-law/<path>` 

* if the path ends in `/xxx/` then strip that
* add `/akn/` at the start and redirect